lib/Backend/Lower.cpp

```diff
@@ -3833,16 +3835,18 @@ Lowerer::GenerateProfiledNewScObjArrayFastPath(IR::Instr *instr, Js::ArrayCallSi
     Func * func = this->m_func;
     IR::LabelInstr * helperLabel = IR::LabelInstr::New(Js::OpCode::Label, func, true);
     uint32 size = length;
+    bool containMissingItems = size != 0;
```

Comment:
Or put parentheses around the compare expression?

---
In reply to: [71740566](https://github.com/Microsoft/ChakraCore/pull/1304#discussion_r71740566) [](ancestors = 71740566,71650529)